xml/System.Configuration.Internal/DelegatingConfigHost.xml

```diff
@@ -829,8 +831,9 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Determines if the delegated host's sections belong to <c>System.Net</c> and the configuration file for the application is downloaded via HTTP using <see cref="T:System.Net.WebClient" />.</summary>
```

Comment:
I am not really sure about this property summary. It was not documented in triple slash, the PR did not describe it and there's no API proposal explaining it. But I found a comment where this property was being used, describing its purpose:
https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/ClientConfigurationSystem.cs#L75

Reply:
It is meant to tell you whether or not the application config file is coming from a non-local URI (i.e. `http://`) or is a local file. That is all I would say here.

Reply:
Okay thanks. I'll use your suggested text:
```suggestion
        <summary>Determines if the application configuration file comes from a non-local URI (that is, <c>http://<c>) or is a local file.</summary>
```